kind/kind.sh

```diff
@@ -99,28 +68,27 @@ function _run_kind_registry() {
     $CTR_CMD run \
         -d --restart=always \
         -p "127.0.0.1:${REGISTRY_PORT}:5000" \
+	--network "${KIND_DEFAULT_NETWORK}" \
         --name "${KIND_REGISTRY_NAME}" \
         registry:2
-    # connect the registry to the cluster network if not already connected
-    $CTR_CMD network connect "${KIND_DEFAULT_NETWORK}" "${KIND_REGISTRY_NAME}" || true
+
+    # Document the local registry
+    # https://github.com/kubernetes/enhancements/tree/master/keps/sig-cluster-lifecycle/generic/1755-communicating-a-local-registry
+
     kubectl apply -f "${KIND_DIR}"/local-registry.yml
 }
 
 function _kind_up() {
-    _fetch_kind
     _prepare_config
     _setup_kind
+    _run_kind_registry
```

Comment:
I think the function should adhere to single-responsiblity principle and only setup kind, not the registry. Use a different function (like it was earlier) to setup registry.

Reply:
Yeah, the reason why I added `_run_kind_registry` into `kind_up` is just to let the `kind` code in current common.sh align with  behavior of `microshift` code.
For current diff,  `_setup_kind` and `_run_kind_registry` are two steps for `_kind_up`, each has single-responsibility already; In common.sh, both `kind` and `microshift` cluster setup will have two similar steps: `_xxx_up` and `_wait_for_clusterReady`, not like current 3 steps for `kind` and 2 steps for `microshift` :-)

Reply:
Speaking of which could this change wait for #31 which does a major overhaul of the scripts merge?